Adding reshape_bondorder_lists_cutoff

Ishikawa-TohokuU opened the pull request:
Simulationdat.pyに、bondorder_list中のcut_off未満の行を削除する関数(reshape_bondorder_list_cutoff)を追加。
Simulationdats.pyに、bondorder_lists中のcut_off未満の行を削除する関数(reshape_bondorder_lists_cutoff)を追加。

Ishikawa-TohokuU committed:
Adding reshape_bondorder_list_cutoff

bondorder_listの中でcut_off未満の行を削除する関数を追加

Ishikawa-TohokuU committed:
Add reshape_bondorder_lists_cutoff

reshape_bondorder_lists_cutoffの追加

kudoryutaro reviewed (changes requested):
kudocop内の原子のidx=Laich内の原子のid-1で統一しているのでidではなくidxでお願いします。id -> atom_idx, connect_id -> connected_atom_idxなど。

docstringを書いてください。参考↓
https://qiita.com/simonritchie/items/49e0813508cad4876b5a#%E6%9B%B8%E3%81%8D%E6%96%B9

Ishikawa-TohokuU committed:
Fixing reshape_bondorder_list_cutoff

■
id -> atom_idx
connect_id -> connected_atom_row
connectの方は原子IDでなく単に配列内での順番を表しているため、このように修正。

■
docstringを追加。

Ishikawa-TohokuU commented:
修正しました
